src/main/java/edu/harvard/iq/dataverse/mydata/Pager.java

```diff
@@ -32,6 +35,7 @@
     
     /* inputs */
     public int numResults;
+    public String numResultsString;
```

Comment:
Should the pager continue to live in the "mydata" package? It's now apparently being using in non-mydata contexts? Should the package be "pager" or something?